content/datasets/unr_idd.md

```diff
@@ -0,0 +1,95 @@
+---
+title: UNR-IDD
+---
+
+- [Overview](#overview)
+- [Environment](#environment)
+- [Activity](#activity)
+- [Contained Data](#contained-data)
+- [Papers](#papers)
+- [Links](#links)
+- [Data Examples](#data-examples)
+
+| <!-- -->                 | <!-- -->                                                                                   |
+| ------------------------ | ------------------------------------------------------------------------------------------ |
+| **Network Data Source**  | Port statistics, selected flow features                                                    |
+| **Network Data Labeled** | Yes                                                                                        |
+| **Host Data Source**     | -                                                                                          |
+| **Host Data Labeled**    | -                                                                                          |
+|                          |                                                                                            |
+| **Overall Setting**      | Miscellaneous                                                                              |
+| **OS Types**             | Linux                                                                                      |
+| **Number of Machines**   | 10 virtual hosts, 12 switches                                                              |
+| **Total Runtime**        | n/a                                                                                        |
+| **Year of Collection**   | 2023                                                                                       |
+| **Attack Categories**    | TCP-SYN flood<br>Port scan<br>Flow table overflow<br>Backhole<br>Diversion                 |
```

Comment:
Blackhole?

Reply:
What they described as

> manipulates flow rules so that traffic is silently dropped or rerouted to unintended paths

They also called this "Diversion". Blackhole might not be the most common term, we could also just go with Diversion or "Traffic Diversion"?

Reply:
ah whoops :T